libqrexec/Makefile

```diff
@@ -1,13 +1,22 @@
 CC=gcc
-override CFLAGS:=-I. -g -O2 -Wall -Wextra -Werror -pie -fPIC `pkg-config --cflags vchan-$(BACKEND_VMM)`$(CFLAGS)
+DISTRO:=$(shell lsb_release -is)
+override QUBES_CFLAGS := -I. -I../libqrexec -g -O2 -Wall -Wextra -Werror \
+   `pkg-config --cflags vchan-$(BACKEND_VMM)` -fstack-protector \
+   -D_FORTIFY_SOURCE=2 -fstack-protector-strong -fPIC \
+   $(CFLAGS)
+override LDFLAGS += -pie -Wl,-z,relro,-z,now -shared
+override LDLIBS=`pkg-config --libs vchan-$(BACKEND_VMM)` -lqrexec-utils -lpam
```

Comment:
A better idea is to use variables other than `LDFLAGS` and `LDLIBS`.

Reply:
But why? Isn't that doing exactly the opposite of what you've just fixed, i.e. allowing to adjust build flags externally?
Please don't do workarounds for weird/broken build environments, especially when it may break legitimate use cases. See [Principle of least surprise](https://en.wikipedia.org/w/index.php?title=Principle_of_least_surprise).

Reply:
Typical Autotools practice is to use something like `$(AM_CFLAGS) $(CFLAGS)`.

Reply:
Ok, then use `$(QUBES_CFLAGS) $(CFLAGS)` in recipe below, but drop `override`.

Reply:
Makes sense.  Also, I can’t do a `git pull` over SSH; it hangs.

Reply:
Do you use this for ssh-agent already?

Reply:
That would explain it :+1:

Reply:
That said, the problem seemed oddly specific to this branch, for some reason.